### PR TITLE
Add Shared host GreenGeeks and CloudAccess.net

### PR DESCRIPTION
**Pull request by photodude**
Add Shared hosts GreenGeeks and CloudAccess.net

### CODE REVIEW

File: _data/hosts.yml [hunk 1]
```diff
@@ -130,6 +130,39 @@
             version: 5.6.11
             semver: 5.6.11
 -
+    name: CloudAccess
+    url: 'http://www.cloudaccess.net/'
+    type: shared
+    php70: 0
```
Comment (philsturgeon) [inline _data/hosts.yml]: This key doesn't need to be there. Can you remove it here, and kill off any other host that has it? Mobile :)

Comment (photodude) [inline _data/hosts.yml]: Which key? do you mean the `php70: 0`?

**Comment by photodude** [comment]
Killed off the `php70: 0` key...

**Comment by philsturgeon** [comment]
Thank you so much!

**Comment by photodude** [comment]
You are welcome, Thank you.

**Pull request merged**